src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,474 @@
+    area_of_interest : gpd.GeoDataFrame
+        A GeoDataFrame polygon specifying the area of interest to retrieve buildings in.
+    antecedent_dry_days: float
+        The number of dry days between rainfall events.
+    average_rain_intensity: float
+        The intensity of the rainfall event in mm/h.
+    event_duration: float
+        The number of hours of the rainfall event.
+    rainfall_ph: float
+        The pH level of the rainfall, a measure of acidity.
+
+    Returns
+    -------
+    None
+        This function does not return any value.
+    """
+    # building_data = []
+    # TODO: Get these values from a dataset
+    all_buildings = get_building_information(engine, area_of_interest)
+    all_roads = get_road_information(engine, area_of_interest)
+
+    print(all_buildings)
+
+    print("START LOOPING...")
+    # Run through each building and calculate TSS, total metal loads, and dissolved metal loads
+    # TODO: change forloop to something meaningful. For the time being, it is simply a placeholder.
+    for index, row in all_buildings.iterrows():
+        surface_area = float(row["SurfaceArea"])
+        surface_type = row["SurfaceType"]
+        curr_tss = compute_tss_roof_road(surface_area=surface_area, antecedent_dry_days=antecedent_dry_days,
+                                         average_rain_intensity=average_rain_intensity, event_duration=event_duration,
+                                         surface_type=surface_type)
+
+        curr_total_copper, curr_total_zinc = total_metal_load_roof(surface_area=surface_area,
+                                                                   antecedent_dry_days=antecedent_dry_days,
+                                                                   average_rain_intensity=average_rain_intensity,
+                                                                   event_duration=event_duration,
+                                                                   rainfall_ph=rainfall_ph, surface_type=surface_type)
+        curr_dissolved_copper, curr_dissolved_zinc = dissolved_metal_load(total_copper_load=curr_total_copper,
+                                                                          total_zinc_load=curr_total_zinc,
+                                                                          surface_type=surface_type)
+        all_buildings.loc[index] = [index, surface_area, surface_type, curr_tss,
+                                    curr_total_copper, curr_total_zinc, curr_dissolved_copper, curr_dissolved_zinc]
+        # print(all_buildings.loc[{'row': i, 'col': "TSS"}])
+        # all_buildings.loc[{'row': i, 'col': 'TSS'}] = curr_tss
+        # all_buildings.loc[{'row': i, 'col': 'TCu'}] = curr_total_copper
+        # all_buildings.loc[{'row': i, 'col': 'TZn'}] = curr_total_zinc
+        # all_buildings.loc[{'row': i, 'col': 'DCu'}] = curr_dissolved_copper
+        # all_buildings.loc[{'row': i, 'col': 'DZn'}] = curr_dissolved_zinc

```

Comment:
Are the commented blocks of code still relevant? It's unclear to anyone else looking at the code. Remove them if possible to prevent code rot, since we can always get them back from the git history. If they are going to be used then add a todo comment